src/compiler/binder.ts

```diff
@@ -1993,16 +1997,33 @@ namespace ts {
                 }
 
                 for (const tag of jsDoc.tags) {
-                    if (tag.kind === SyntaxKind.JSDocTypedefTag) {
+                    if (isJSDocTypeAlias(tag)) {
+                        const savedParent = parent;
+                        parent = jsDoc;
+                        bind(tag);
+                        parent = savedParent;
+                    }
+                    // Bind template tags that have a typedef or callback tag in the same comment.
+                    // The typedef/callback tag is the container of the template.
+                    const alias = getTypeAliasForJSDocTemplateTag(tag, jsDoc.tags);

```

Comment:
Why are we doing this? is not a typeAlais always introducing its own scope? at least for `@property` and `@tempate`?